code-injection-analysis/ErtixNodes_panel/old.js

```
@@ -0,0 +1,538 @@
+    if (!dbUser) return res.json({ ok: false });
+
+    if (!dbUser.serverLimit) {
+        dbUser.serverLimit = 1;
+        await dbUser.save();
+    }
+
+    var srv = await db.Server.find({
+        userID: req.session.userID
+    });
+
+    if (srv.length >= dbUser.serverLimit) {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)